index.html

```diff
@@ -49,7 +49,6 @@ <h1>Im Lucca Glad to see you</h1>
         <h5>Lets Connect</h5>
 
     </footer>
-    
     <div>
         <ul>
```

Comment:
You did a great job making it an unordered list, however, each list item should also be wrapped in  `a` tag. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons using `href`.

Reply:
Implemented 👍